lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_page.inc

```diff
@@ -234,17 +234,6 @@ function paraneue_dosomething_preprocess_page(&$vars) {
     'terms' => dosomething_user_get_link_terms_service(),
   );
 
-  // Footer affiliate logo.
-  $footer_logo_enabled = theme_get_setting('footer_affiliate_logo');
```

Comment:
I _think_ so (although it's easy to remove the commit). Wasn't that only used for 💀 international affiliate sites?